app-backend/db/src/main/scala/AnnotationLabelDao.scala

```diff
@@ -286,4 +286,48 @@ object AnnotationLabelDao extends Dao[AnnotationLabelWithClasses] {
       Some(fr"session_id = ${sessionId}")
     )).update.run
   }
+
+  def toggleByLabelId(
```

Comment:
A nit -- I think this could be named better as:

```suggestion
  def toggleActiveByLabelId(
```

We have two things that could conceivably be toggle-able on labels (`status` and `is_active`) and I think adding the specific thing we're toggling will be a little clearer

Reply:
@jisantuc agreed -- will address in a new commit.